core/sections/clause_8_ogc-process-description.adoc

```diff
@@ -123,6 +123,9 @@ The <<jsonschemavalidation,JSON Schema specification>> defines a https://json-sc
 |http://www.opengis.net/def/format/ogcapi-processes/0/cql2-text-filter |cql2-text |Indicates that the string value is a filter expression encoded using the text representation of the https://docs.ogc.org/is/21-065r2/21-065r2.html[Common Query Language (CQL2)].
 |http://www.opengis.net/def/format/ogcapi-processes/0/cql2-json-filter |cql2-json |Indicates that the string value is a filter expression encoded using the JSON representation of the https://docs.ogc.org/is/21-065r2/21-065r2.html[Common Query Language (CQL2)].
 |http://www.opengis.net/def/format/ogcapi-processes/0/collection-id |collection-id |Indicates that the string value is a https://docs.ogc.org/DRAFTS/20-024.html#collection-description[collection identifier] (e.g. a feature collection identifier).
+|http://www.opengis.net/def/format/ogcapi-processes/0/stac-collection |stac-collection |Indicates that the string value is a https://github.com/radiantearth/stac-spec/blob/master/collection-spec/collection-spec.md[STAC collection].
```

Comment:
I got yet one more to propose while we are at it:  `stac-items`
Just encountered that case earlier.

The idea being, that you can request the STAC Items themselves as `application/geo+json`. 
Because STAC Items have nested media-types as Assets, doing a request with only `type` is ambiguous (is it the items or the assets they describe?) 
ie: 

```yaml
some-input:
  collection: https://server.com/stac/api/v1/collection/something
  type: application/geo+json  # Should this be the Items FeatureCollection, or a Vector file found in Assets?
  format: stac-items  # specifically asking for the items, needed to disambiguate from OA Features
```

```yaml
some-input:
  collection: https://server.com/stac/api/v1/collection/something
  type: image/tiff; application=geotiff; profile=cloud-optimized
  format: stac-collection  # STAC Assets matching the COG media-type
  # but why 'stac-collection'? -> properties to find them (therefore, not a OA Coverages)
  filter: properties.eo:cloud_cover > 0.5
```

Reply:
Relates to https://github.com/crim-ca/weaver/pull/770

Reply:
In STAC thats named ItemCollection, so maybe use stac-itemcollection instead?

Reply:
And what about a single item or a catalog?

Reply:
There's nothing to align to right now, I think. 
There's just different specifications which for all there could be a usecase to pass them to processes. We discussed that already in another issue, which listed all potential options, I think.

- stac -> The union of all of the types below, i.e. any STAC entity
- stac-item -> STAC Item Spec (static or /collections/:id/items/:id)
- stac-collection -> STAC Collection Spec (static or /collections/:id)
- stac-catalog -> STAC Catalog spec (static or API landing page)
- stac-itemcollections -> ItemCollection fragment in STAC API (/collections/:id/item or /search)
- (stac-api -> a STAC API landing page, could potentially be captured by stac-catalog?)
- (stac-collectioncollection -> What you get returned in /collections - but I don't have a usecase for this, so can be omitted)
-